src/lib.rs

```diff
@@ -157,9 +162,21 @@ impl CheckM2QualityReport {
 
         for result in rdr.records() {
             let res = result.expect("Parsing error in CheckM qualities file");
-            let completeness: f32 = res[1]
-                .parse::<f32>()
-                .expect("Error parsing completeness in checkm tab table");
+
+            let completeness: f32 = if completeness_choice {
+                if &res[4] == "Neural Network (Specific Model)" {
+                    // Completeness_Specific
+                    &res[3]
+                } else {
```

Comment:
Given there seems to be changing formats, can you check with an else if, and then error on everything else?

Maybe also more rusty to use match

Reply:
How is that?